pyramid_tm/__init__.py

```diff
@@ -1,15 +1,21 @@
 import sys
 import transaction
 
+from pyramid.httpexceptions import HTTPNotFound
+from pyramid.interfaces import IRequestExtensions
+from pyramid.interfaces import IRequestFactory
+from pyramid.request import Request, apply_request_extensions
 from pyramid.settings import asbool
-from pyramid.util import DottedNameResolver
+from pyramid.threadlocal import manager as request_manager
```

Comment:
I know `pyramid_tm` is special, but this is not public API, by using it here are we not de facto making it a public API?